tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -0,0 +1,411 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import ocp, constants, defaults
+from ocs_ci.framework.testlib import workloads, E2ETest, ignore_leftovers, tier4
+from ocs_ci.ocs.resources import pod
+from tests.helpers import wait_for_resource_state, default_storage_class
+from tests.disruption_helpers import Disruptions
+from tests.sanity_helpers import Sanity
+from ocs_ci.ocs.monitoring import (
+    check_pvcdata_collected_on_prometheus,
+    check_ceph_health_status_metrics_on_prometheus,
+    prometheus_health_check
+)
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status, get_node_objs,
+    get_typed_nodes, drain_nodes, schedule_nodes
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed
+
+log = logging.getLogger(__name__)
+
+
+@retry(AssertionError, tries=30, delay=3, backoff=1)
+def wait_to_update_mgrpod_info_prometheus_pod():
+
+    log.info(
+        f"Verifying ceph health status metrics is updated after rebooting the node"
+    )
+    ocp_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    mgr_pod = (
+        ocp_obj.get(selector=constants.MGR_APP_LABEL).get('items')[0].get('metadata').get('name')
+    )
+    assert check_ceph_health_status_metrics_on_prometheus(mgr_pod=mgr_pod), (
+        f"Ceph health status metrics are not updated after the rebooting node where the mgr running"
+    )
+    log.info("Ceph health status metrics is updated")
+
+
+@retry((CommandFailed, TimeoutError), tries=10, delay=3, backoff=1)
+def wait_for_master_node_to_be_running_state():
+    """
+    Waits for the all the nodes to be in running state
+    """
+    wait_for_nodes_status(timeout=900)
+
+
+@tier4
```

Comment:
Should be workloads tier

Reply:
@ebenahar Are we not segregating the workloads into tiers ?